pod/pod-api-public.yaml

```diff
@@ -4522,7 +4522,71 @@ paths:
         '500':
           description: 'Server error, see response body for further details.'
           schema:
-            $ref: '#/definitions/Error'
+            $ref: '#/definitions/Error'
+  '/dataquery/api/v3/messages/thread':
```

Comment:
@symphonydarlys , from what I just discovered, this endpoint should not be documented here since its payload required decryption.
The new sid attribute would only be consumed by bots through the Agent's API.